client/src/qa/components/QuestionAndAnswerEntry.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import QuestionEntry from './QuestionEntry';
+import AnswerEntry from './AnswerEntry';
+
+const QuestionAndAnswerEntry = ({ question, answers }) => {
+  const [slice, setSlice] = useState(2);
+  const initialAnswers = answers.slice(0, slice);
+  const [currentAnswers, setCurrentAnswers] = useState(initialAnswers);
+  const numOfAnswersLeft = answers.length - currentAnswers.length;
+
+  return (
+    <div>
+      <QuestionEntry question={question} />
+      {currentAnswers.map((answer, index) => (
+        <AnswerEntry key={index} answer={answer} />
+      ))}
+      {numOfAnswersLeft
+        && <button onClick={() => {
+          // setCurrentAnswers(prevAnswers => ...prevAnswers, answers)
```

Comment:
Overall I like the use of hooks, it's simple and makes sense. I can see where you're going with your logic.